components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/internal/APIManagerComponent.java

```diff
@@ -986,6 +987,19 @@ protected void addOrganizationResolver(OrganizationResolver resolver) {
     protected void removeOrganizationResolver(OrganizationResolver resolver) {
         ServiceReferenceHolder.getInstance().setOrganizationResolver(null);
     }
-    
+
+    @Reference(
+            name = "resourceQuotaLimiter.service",
+            service = ResourceQuotaLimiter.class,
+            cardinality = ReferenceCardinality.OPTIONAL,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "removeQuotaLimiter")
```

Comment:
Lets have the closing bracket in next line if possible

Reply:
All existing previous components follow the above formatting style.